ddtrace/internal/core/__init__.py

```diff
@@ -273,10 +273,24 @@ def root(self):
             current = current.parent
         return current
 
+    @property
+    def span(self) -> "Span":
+        if self._inner_span is None:
+            raise ValueError("No span set on ExecutionContext")
```

Comment:
Are we catching this anywhere? I think we want to avoid raising errors all the way up if possible right?

Reply:
Oh in get_span, got it.